db/migrate/20170607101716_create_themes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateThemes < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
Sounds good. I really like the idea of using `Group` as it would mean things could be more flexible in the future without us having to make structural changes to the database.